src/utils/rest-api.js

```diff
@@ -710,6 +710,25 @@ export function updateTreeNode(node) {
     );
 }
 
+export function deleteModification(studyUuid, node, modificationUuid) {
+    const modificationDeleteUrl =
+        PREFIX_STUDY_QUERIES +
+        '/v1/studies/' +
+        studyUuid +
```

Comment:
This came from a `decodeURIComponent` then you should `encodeURIComponent` as well for studyUuid I think.

Reply:
done